courses/machine_learning/deepdive2/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,1751 @@
+{
```

Comment:
<p>I believe that for this to work correctly the 'data' directory needs to exist. </p><p>You could do this with !mkdir data in a cell above. Or (preferable) you could do os.mkdir('data') in this cell before those three lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>